server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CachedBlobStore.java

```diff
@@ -96,22 +98,32 @@ private ReadAheadInputStream(PushbackInputStream in, Optional<byte[]> firstBytes
             this.firstBytes = firstBytes;
             this.hasMore = hasMore;
         }
-
     }
 
     public static final String BACKEND = "blobStoreBackend";
+    public static final String BLOBSTORE_CACHED_LATENCY_METRIC_NAME = "blobstoreCachedLatency";
+    public static final String BLOBSTORE_CACHED_HIT_COUNT_METRIC_NAME = "blobstoreCachedHit";
```

Comment:
```suggestion
    public static final String BLOBSTORE_CACHED_HIT_COUNT_METRIC_NAME = "blobstoreCachedHits";
```

Reply:
good catch

Reply:
What we are measuring latency for is a cache for the blobStore hence the name `blobStoreCache` seems more appropriate.